<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Trajectory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="trajectoryTable">
     <property name="columnCount">
      <number>7</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="velocityLabel">
         <property name="text">
          <string>Velocity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="velocitySlider">
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="velocityValue">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="acelerationLabel">
         <property name="text">
          <string>Aceleration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="acelerationSlider">
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="acelerationValue">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="loggingCheck">
     <property name="text">
      <string>logging</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sendButton">
     <property name="text">
      <string>Send to robot</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
